dlp/snippets/deid.py

```diff
@@ -937,6 +937,79 @@ def deidentify_with_replace_infotype(project, item, info_types):
 # [END dlp_deidentify_replace_infotype]
 
 
+# [START dlp_deidentify_simple_word_list]
+def deidentify_with_simple_word_list(
+    project,
+    input_str,
+    info_type,
+    word_list
+):
+    """Uses the Data Loss Prevention API to de-identify sensitive data in a
+      string by matching against custom word list.
+
+    Args:
+        project: The Google Cloud project id to use as a parent resource.
+        input_str: The string to deidentify (will be treated as text).
+        info_type: A string representing info type to look for.
+            A full list of info type categories can be fetched from the API.
+        word_list: The list of strings to match against.
+
+    Returns:
+          None; the response from the API is printed to the terminal.
+    """
+    # Import the client library
+    import google.cloud.dlp
+
+    # Instantiate a client.
+    dlp = google.cloud.dlp_v2.DlpServiceClient()
+
+    # Prepare custom_info_types by parsing word lists
+    word_list = {"words": word_list}
+    custom_info_types = [
+        {
+            "info_type": {"name": info_type},
+            "dictionary": {"word_list": word_list}
+        }
+    ]
+
+    # Construct the configuration dictionary
+    inspect_config = {
+        "custom_info_types": custom_info_types,
+    }
+
+    # Construct deidentify configuration dictionary
+    deidentify_config = {
+        "info_type_transformations": {
+            "transformations": [
+                {
+                    "primitive_transformation": {"replace_with_info_type_config": {}}
+                }
+            ]
+        }
+    }
+
+    # Construct the `item`.
+    item = {"value": input_str}
+
+    # Convert the project id into a full resource id.
+    parent = f"projects/{project}"
+
+    # Call the API
+    response = dlp.deidentify_content(
+        request={
+            "parent": parent,
+            "deidentify_config": deidentify_config,
+            "inspect_config": inspect_config,
+            "item": item,
+        }
+    )
+
+    print("De-identified Content: {}".format(response.item.value))
```

Comment:
Use f-string instead of `format`, unless it is used with logging libraries.
```suggestion
    print(f"De-identified Content: {response.item.value}")
```

Reply:
yes, updated that.